Cargo.toml

```diff
@@ -1,14 +1,17 @@
 [package]
-name = "imap-proto"
-version = "0.1.0"
 authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
+categories = ["authentication", "email", "encoding", "network-programming", "parser-implementations"]
 description = "IMAP protocol parser and data structures"
 documentation = "https://docs.rs/imap-proto"
-keywords = ["imap"]
-categories = ["email"]
 homepage = "https://github.com/djc/imap-proto"
+keywords = ["IMAP", "e-mail", "email"]
```

Comment:
Oh, I also don't want to add `e-mail` here, seeing as there's only one package that currently has `e-mail` vs 27 that have `email`. And it looks like crates.io treats keywords as all-lowercase, so I'd want `imap` here rather than `IMAP`.

Reply:
Okay! I find it sad though to see programmers misspell these words, which are so important for discoverabilty ..